### ISSUE TITLE
Edit/Delete of task 

### ISSUE THREAD

**Issue by gizbo** [angular-gantt/angular-gantt#334]
Hello,
   I'm looking for the best way to delete and edit task that have been draw.  Right now I'm experimenting with 'api.directives.on.new' functionality to dynamically add a trash and a pencil icon on the task by hovering on them. The idea is the bind a click event  on those icon. In the case of the delete I would call '$scope.api.data.remove(directiveScope.task.model);'. For the edit I'd like to display a model window with the task model field.  Is this the proper approach?  
Thanks so much,
Yves

**Comment by Toilal** [comment]
I think it's the proper approach yes. Use API.

You can also remove data directly from your data object model, it should 
work (2 way binding).

**Comment by Toilal** [comment]
@gizbo I read too quickly. If you use master version, you can use [task-content](http://angular-gantt.readthedocs.org/en/latest/configuration/attributes/#task-content) attribute to customize the content template of each task. 

There are many new attributes and properties named `content` in many part of the component to use custom HTML inside default templates. It allow to add icon or custom HTML in key parts, instead of only using strings.

**Comment by Toilal** [comment]
See the current demo, there are some custom icons in tasks and rows.

**Comment by gizbo** [comment]
Hi Toilal,
Thanks for your quick replies. Listen, I cloned the master branch and I'm running the demo unfortunately I don't see the icon being displayed, I do see the tasks contain a content field with html markup for an icon but for some reason it does't display, what am I doing wrong? Also I had to remove the all point-events from the css to get the draw working (got this from one of your other topic comments). Would it be possible to create a event that would fire right after to user lets go to mouse when drawing a new task? I'd like to display a modal windows to capture the task name, color etc. that way? 
Thanks so much for help,
Yves
